tests/xbot/common/injection/PersonalComputerDatabaseStorage.java

```diff
@@ -0,0 +1,13 @@
+package xbot.common.injection;
+
+import xbot.common.properties.DatabaseStorageBase;
+
+public class PersonalComputerDatabaseStorage extends DatabaseStorageBase {
+
+	public static String testFolder = "./TeamDatabase";
```

Comment:
This totally works, but consider playing with the guice named parameters to be able to provide this string in the binding module instead of needing classes: https://github.com/google/guice/wiki/BindingAnnotations#named